static/src/stylesheets/module/commercial/_advert.scss

```diff
@@ -126,9 +126,9 @@
     display: flex;
     align-items: stretch;
 
-    > * {
-        width: 100%;
-    }
+    // > * {
+    //     width: 100%;
+    // }
```

Comment:
hmm, yeah presumably you are referring to the layout of the badge? If yes, then fair enough. Perhaps have a quick look on IE11 if it has any effect.